serverless/pages/spaces.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: /serverless/spaces
+title: Spaces
+description: Organize your project and objects into multiple spaces
+---
+
+Spaces enable you to organize your dashboards and other saved
+objects into meaningful categories. Once inside a space, you see only
+the dashboards and saved objects that belong to that space.
+
+When you create and enter a new project, you're using the default space of that project.
+
+You can identify the space you're in or switch to a different space from the header.
+
+<DocImage size="l" url="../images/space-breadcrumb.png" alt="Space breadcrumb" />
+
+You can view and manage the spaces of a project from the **Spaces** page in **Management**.
+
+## Required permissions
+
+You must have the admin role on the project to manage its **Spaces**.
+
+## Create or edit a space
+
+You can create as many spaces as you need. 
+
+1. Click *Create space* or select the space you want to edit.
+
+2. Provide:
+    - A meaningful name and description for the space.
+    - A URL identifier. The URL identifier is a short text string that becomes part of the ((kib)) URL. ((kib)) suggests a URL identifier based on the name of your space, but you can customize the identifier to your liking. You cannot change the space identifier later.
+
+3. Customize the avatar of the space to your liking.
+
+4. Save the space.
+
+((kib)) also has an [API](https://www.elastic.co/docs/api/doc/serverless)
+if you prefer to create spaces programmatically.
+
+<DocIf condition={"((serverlessCustomRoles))" === "true"}>
+## Customize access to space
+
+As an administrator, you can define custom roles with specific access to certain spaces and features in a project. Refer to <DocLink slug="/serverless/custom-roles"/>.
+</DocIf>
```

Comment:
Does this `DocIf` block pertain to specific project types, or is this check for the entire environment? I ask because custom roles will not be available for Observability projects right away. Only Security and Search projects will have this capability for GA.

Reply:
I added this DocIf because the rest of the Custom roles docs are currently behind that same flag.

To indicate availability per project type, we use badges, which I'll add in a next commit. Thanks for the information!

Reply:
Added badges